login-workflow/src/new-architecture/screens/AccountDetailsScreen/AccountDetailsScreen.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import { AccountDetailsScreenBase, AccountDetailsScreenProps } from '../AccountDetailsScreen';
+import { useLanguageLocale } from '../../hooks';
+import { useRegistrationWorkflowContext } from '../../contexts';
+
+type AccountDetailsFullScreenProps = AccountDetailsScreenProps & {
+    title?: string;
+    instructions?: string;
+};
+
+export const AccountDetailsScreen: React.FC<AccountDetailsFullScreenProps> = (props) => {
+    const { t } = useLanguageLocale();
+    const regWorkflow = useRegistrationWorkflowContext();
+    const { nextScreen, previousScreen, screenData } = regWorkflow;
+    const [firstName, setFirstName] = useState(screenData.AccountDetails.firstName ?? '');
+    const [lastName, setLastName] = useState(screenData.AccountDetails.lastName ?? '');
+
+    const {
+        title = t('bluiRegistration:REGISTRATION.STEPS.ACCOUNT_DETAILS'),
+        instructions = t('bluiRegistration:REGISTRATION.INSTRUCTIONS.ACCOUNT_DETAILS'),
+        firstNameLabel = t('bluiCommon:FORMS.FIRST_NAME'),
+        /* eslint-disable @typescript-eslint/no-unused-vars */
+        initialFirstName,
+        initialLastName,
+        lastNameLabel = t('bluiCommon:FORMS.LAST_NAME'),
+        firstNameTextFieldProps,
+        lastNameTextFieldProps,
+        firstNameValidator = (name: string): boolean | string => {
+            if (name?.length > 2) {
+                setFirstName(name);
+                return true;
+            }
+            return t('bluiCommon:FORMS.FIRST_NAME_LENGTH_ERROR');
+        },
+        lastNameValidator = (name: string): boolean | string => {
+            if (name?.length > 2) {
+                setLastName(name);
+                return true;
+            }
+            return t('bluiCommon:FORMS.LAST_NAME_LENGTH_ERROR');
+        },
+        sx,
```

Comment:
You can remove `sx` prop from here as you are not passing any styles inside of it